--- otp_src_R15B/erts/emulator/Makefile.in.orig	2013-10-31 01:20:17.724541700 -0700
+++ otp_src_R15B/erts/emulator/Makefile.in	2013-10-31 01:21:48.531959267 -0700
@@ -855,9 +855,10 @@
 	$(CC) $(subst O2,O3, $(CFLAGS)) $(INCLUDES) -c $< -o $@
 
 $(BINDIR)/hipe_mkliterals$(TF_MARKER):	$(OBJDIR)/hipe_mkliterals.o
-	$(CC) $(CFLAGS) $(INCLUDES) -o $@ $<
+	$(BUILD_CC) $(CFLAGS) $(INCLUDES) -o $@ hipe/hipe_mkliterals.c
 
 $(OBJDIR)/hipe_mkliterals.o:	$(TTF_DIR)/hipe_x86_asm.h $(TTF_DIR)/hipe_ppc_asm.h $(TTF_DIR)/beam_opcodes.h
+	$(BUILD_CC) $(CFLAGS) $(INCLUDES) -c hipe/hipe_mkliterals.c
 
 $(TTF_DIR)/hipe_literals.h:	$(BINDIR)/hipe_mkliterals$(TF_MARKER)
 	$(BINDIR)/hipe_mkliterals$(TF_MARKER) -c > $@
