--- otp_src_R15B/lib/erl_interface/src/Makefile.in.orig	2013-10-31 18:40:56.781483110 -0700
+++ otp_src_R15B/lib/erl_interface/src/Makefile.in	2013-10-31 18:42:39.708832458 -0700
@@ -689,11 +689,11 @@
 ifdef THR_DEFS
 $(ERL_CALL): $(ERLCALL) ../include/ei.h $(MT_EILIB)
 	$(PURIFY) $(CC) $(PROG_CFLAGS) $(THR_DEFS) $(LDFLAGS) -o $@ $(ERLCALL) \
-		-L$(OBJDIR) -lei $(THR_LIBS) $(LIBS)
+		-L$(OBJDIR) -lei $(THR_LIBS) $(LIBS) $(LDFLAGS)
 else
 $(ERL_CALL): $(ERLCALL) ../include/ei.h $(ST_EILIB)
 	$(PURIFY) $(CC) $(PROG_CFLAGS) $(LDFLAGS) -o $@ $(ERLCALL) \
-		-L$(OBJDIR) -lei $(LIBS)
+		-L$(OBJDIR) -lei $(LIBS) $(LDFLAGS)
 endif
 endif
 endif
